ddp-workspace/projects/ddp-sdk/src/lib/components/tooltip.component.ts

```diff
@@ -13,10 +18,28 @@ import { ConfigurationService } from '../services/configuration.service';
              (click)="tooltip.toggle()"
              #tooltip="matTooltip">`
 })
-export class TooltipComponent {
+export class TooltipComponent implements OnInit, OnDestroy {
     @Input() icon = this.config.tooltipIconUrl;
     @Input() text: string;
     @Input() position = 'right';
+    destroyed = new Subject<void>();
 
-    constructor(@Inject('ddp.config') private config: ConfigurationService) { }
+    constructor(@Inject('ddp.config') private config: ConfigurationService, private breakpointObserver: BreakpointObserver) { }
+    ngOnInit(): void {
+        const initialPosition = this.position;
+        this.breakpointObserver.observe([
+            Breakpoints.XSmall,
+            Breakpoints.Small,
+            Breakpoints.Medium,
+            Breakpoints.Large,
+            Breakpoints.XLarge
+        ]).pipe(takeUntil(this.destroyed))
+            .subscribe(() => {
+                this.position = this.breakpointObserver.isMatched(Breakpoints.XSmall) ? 'below' : initialPosition;
+        });
```

Comment:
Seems like we can simplify it a bit, like:
```
this.breakpointObserver.observe([Breakpoints.XSmall])
  .pipe(takeUntil(this.destroyed))
  .subscribe((result: BreakpointState) => {
     if (result.matches) {
       this.position = 'below'; 
     }    
  });
```

Have you tried that?

Reply:
I haven't tried and it actually makes more sense since on a real device it is not possible to move from mobile to a larger screen. It's done.